agent/src/AgentFixupControls.ts

```diff
@@ -51,6 +51,7 @@ export class AgentFixupControls implements FixupControlApplicator {
             state: task.state,
             error: errorToCodyError(task.error),
             selectionRange: task.selectionRange,
+            instruction: task.instruction.toString().trim()
```

Comment:
```suggestion
            instruction: task.instruction?.trim()
```

✋ q: do we need to add ? to check if the instruction string is undefined before we trim it? 

(Ignore me if there was a check somewhere that I missed 🙈)

Reply:
Why do we need to trim it at all here, shouldn't we do that when it is put into `FixupTask`?
Anyway LGTM

Reply:
good catch; I'll add that.